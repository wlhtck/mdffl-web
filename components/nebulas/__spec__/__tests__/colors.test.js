import {
  primary, secondary, black, grey, white,
} from '../../colors';

describe('components/nebulas/colors', () => {
  it('exports primary', () => {
    expect(primary).toMatchSnapshot();
  });
  it('exports secondary', () => {
    expect(secondary).toMatchSnapshot();
  });
  it('exports black', () => {
    expect(black).toMatchSnapshot();
  });
  it('exports grey', () => {
    expect(grey).toMatchSnapshot();
  });
  it('exports white', () => {
    expect(white).toMatchSnapshot();
  });
});
