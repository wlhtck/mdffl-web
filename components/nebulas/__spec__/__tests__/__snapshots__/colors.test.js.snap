// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/nebulas/colors exports black 1`] = `"#303133"`;

exports[`components/nebulas/colors exports grey 1`] = `"#8D8D8F"`;

exports[`components/nebulas/colors exports primary 1`] = `"#1C104E"`;

exports[`components/nebulas/colors exports secondary 1`] = `"#F5A623"`;

exports[`components/nebulas/colors exports white 1`] = `"#FFFFFF"`;
